__tests__/app.tests.js

```diff
@@ -112,4 +112,72 @@ describe("nc news app", () => {
         });
     });
   });
+  describe("GET /api/articles", () => {
+    test('status: 200, responds with an object containing a key of "articles" and a value of an array of objects', () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body }) => {
+          expect(Object.keys(body)[0]).toBe("articles");
+          expect(body.articles).toBeInstanceOf(Array);
+          expect(body.articles[0]).toBeInstanceOf(Object);
+        });
+    });
+    test("status: 200, responds with an array of article objects with the correct keys", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.articles).toHaveLength(12);
+          body.articles.forEach((article) => {
+            expect(article).toMatchObject({
+              article_id: expect.any(Number),
+              author: expect.any(String),
+              title: expect.any(String),
+              topic: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              article_img_url: expect.any(String),
```

Comment:
no need for the img url :) you might need that later on when you do a single article, but not for the whole lot :)

Reply:
On the trello board instructions is says it should return an article_img_url?